### PR TITLE
added nav active link helper

### PR DESCRIPTION
**Pull request by KwaneleNdenze**
Had to delete the development branch and create a new one. Will create a 3rd branch for future changes to be merged with the develop branch.

### CODE REVIEW

File: app/helpers/nav_active_link_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module NavActiveLinkHelper
+  def get_active_link(path)
+    current_page?(path) ? 'active' : ''
+  end
```
Comment (thatosmk) [inline app/helpers/nav_active_link_helper.rb:4]:
```suggestion
  def active_link_css(path)
    current_page?(path) ? 'active' : ''
  end
```

**Comment by thatosmk** [comment]
you can put this into `Ready for review`, it looks great so far. great work man

File: app/helpers/nav_active_link_helper.rb (hunk 1 above)
Comment (thatosmk) [reply app/helpers/nav_active_link_helper.rb:4]: but you need to write a test for this as well.

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -339,3 +339,6 @@
   </main><!-- End #main -->
 
   
+
+
+<% console %>
```
Comment (thatosmk) [inline app/views/pages/index.html.erb:344]: please remove this

Comment (KwaneleNdenze) [reply app/views/pages/index.html.erb]: Done.

File: spec/helpers/nav_active_link_helper.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+RSpec.describe NavActiveLinkHelper, :type => :helper do
+  describe "#active_link_css" do
+    it "checks if current page is index and makes link active" do
+      path = helper.root_path
+      allow(helper).to receive(:current_page?).with(path).and_return(true)
+      expect(helper.active_link_css(path)).to eq("active")
+
+      allow(helper).to receive(:current_page?).with(path).and_return(false)
+      expect(helper.active_link_css(path)).to eq("")
+    end
```
Comment (thatosmk) [inline spec/helpers/nav_active_link_helper.rb:12]:
```suggestion
    context "when current_page? is true" do
      it do
        path = helper.root_path
        allow(helper).to receive(:current_page?).with(path).and_return(true)
        expect(helper.active_link_css(path)).to eq("active")
      end

      context 'when current_page? is false' do
        it do
          allow(helper).to receive(:current_page?).with(helper.root_path).and_return(false)
          expect(helper.active_link_css(path)).to eq("")
        end
      end
    end
```

Comment (thatosmk) [reply spec/helpers/nav_active_link_helper.rb:12]: try that

File: app/helpers/nav_active_link_helper.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module NavActiveLinkHelper
```
Comment (thatosmk) [inline app/helpers/nav_active_link_helper.rb:1]: also you do not need to create a new helper for this, you can simply add the new method in `ApplicationHelper`.

Look at [this example](https://github.com/forem/forem/blob/main/app/helpers/application_helper.rb)

Comment (thatosmk) [reply app/helpers/nav_active_link_helper.rb:1]: no pressure to do this now, can add it in your next PR

**Review by thatosmk — APPROVED** [review]

**Pull request merged**